src/stories/atoms/dialog.stories.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from 'react';
+
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { Dialog } from '../../atoms';
+import Button from '@mui/material/Button';
+
+const meta: Meta<typeof Dialog> = {
+  title: 'Atoms/Dialog',
+  component: Dialog,
+  parameters: {
+    layout: 'centered',
+  },
+  tags: ['autodocs'],
+  argTypes: {
+    type: {
```

Comment:
If the actions prop won't be allowed to be modified, I consider it would be better to just not show it in the StoryBook with `table: {disable: true}`
![image](https://github.com/user-attachments/assets/85ed3fed-a7a6-40e7-b936-8269c80f0440)